src/pages/BookmarkPage.jsx

```diff
@@ -0,0 +1,83 @@
+import { useEffect, useState } from 'react';
+import BookmarkCard from '../components/BookmarkCard';
+import data from '../mockData';
```

Comment:
Let's but an empty line between local imports and other imports

```suggestion
import { useEffect, useState } from 'react';

import BookmarkCard from '../components/BookmarkCard';
import data from '../mockData';
```